packages/service/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vuesion/service",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
can you revert this line? I will increase the version if I release a new npm version.

BTW: is this change a bc break or can we use the new storybook version with the existing files in the vue-starter?

Reply:
I will revert the version. We can use the new storybook version with the existing files in the vue-starter, its just that the order of story cluster (Design systems, Atoms, molecules and organisms) doesn't seem to be aligned as I was expecting

Reply:
that's ok. I just need to know if I have to release a major version 👍

Reply:
I think I might have misunderstood something. There is also some changes in vue starter for storybook config files for which I will raise a PR in a moment. Fortunately the changes are only in config and not in stories